src/ol_openedx_chat_xblock/ol_openedx_chat_xblock/static/css/ai_chat_xblock.css

```diff
@@ -1,4 +1,21 @@
 .ai-chat-container {
-    min-height: 400px;
-    position: relative;
+  /*
+  * MFE Iframe doesn't have a height set, so ensure a min height.
+  */
+  min-height: 400px;
+  height: calc(100vh - 100px);
```

Comment:
I am not sure, but this is breaking Learning MFE design. Have you tested this change?